visgraph/graphcore.py

```diff
@@ -303,6 +303,14 @@ def delNodeProp(self, node, prop):
                 self.nodeprops[prop].pop(pval,None)
         return pval
 
+    def delNodesProps(self, props=('up','down')):
```

Comment:
These default args are super use case specific and probably exist as a hold-over from the function being within the other module.  props in this case should not be a kwarg.